docs/howto/krustlet-on-wsl2.md

```diff
@@ -0,0 +1,148 @@
+# Running Krustlet on WSL2 with Docker Desktop
+
+This how-to guide demonstrates how to boot a Krustlet node in Docker Desktop for Windows with WSL2 backend.
+
+## Information
+This tutorial will work on current Windows 10 Insider Slow ring and Docker Desktop for Windows stable release.
+
+Concerning Windows, this tutorial should work on the Production ring once it will be available.
+
+Last but not least, this will work on Windows 10 Home edition.
+
+## Prerequisites
+
+You will require a WSL2 distro and Docker Desktop for Windows for this how-to. The WSL2 backend and Kubernetes features will need to be also enabled.
+See the [Docker Desktop for Windows > Getting started > Kubernetes](https://docs.docker.com/docker-for-windows/#kubernetes) howto for more information.
+
+This specific tutorial will be running Krustlet on your WSL2 distro and will explain how to access it from Windows.
+
+## Step 1: Create Certificate
```

Comment:
This entire step is not necessary.

instead, type:
`kubectl config get-contexts`

then `kubectl config set-context docker-desktop`

Reply:
this may be dependent on enabling WSL integration in docker> settings > resources > WSL Integration

Reply:
oh ok, nice, I kept it form the KinD instructions as it was "approximately" the same.
And if we are only using Docker Desktop (`docker` and `kubectl` will be "mounted" from the docker desktop distro), then the context is by default already docker-desktop

Reply:
what I can/will do is set it as optional then